src/xpcc/architecture/platform/devices/avr/atmega1281_2561-none.xml

```diff
@@ -15,6 +15,7 @@
     <pin-count>0</pin-count>
     <header>avr/io.h</header>
     <header>avr/interrupt.h</header>
+    <define>__ATmegaxx0__</define>
```

Comment:
Hm?

In file `iomxx0_1.h` this gets defined already:
```c
#if defined(__AVR_ATmega640__) || defined(__AVR_ATmega1280__) || defined(__AVR_ATmega2560__)
# define __ATmegaxx0__
#elif defined(__AVR_ATmega1281__) || defined(__AVR_ATmega2561__)
# define __ATmegaxx1__
#endif
```
This shouldn't be necessary?
Also, shouldn't this be `__ATmegaxx1__` according to the above logic?